source/MaterialXGenGlsl/GlslSyntax.cpp

```diff
@@ -236,6 +236,16 @@ GlslSyntax::GlslSyntax()
             "struct VDF { vec3 absorption; vec3 scattering; };")
     );
 
+    registerTypeSyntax
+    (
+        Type::ROUGHNESSINFO,
+        std::make_shared<AggregateTypeSyntax>(
+            "roughnessinfo",
+            "roughnessinfo(0.0, 0.0, 0.0, 0.0)",
+            EMPTY_STRING,
+            "struct roughnessinfo { float roughness; float alpha; float alphaX; float alphaY; };")
```

Comment:
Struct contains:

roughness - the original roughness value
alpha - the scalar value used by BSDF's. Equals roughness^2, to achieve a more uniform roughness look over
the parameter range.
alphaX - the X component for anisotropic roughness
alphaY - the Y component for anisotropic roughness